stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/User.kt

```diff
@@ -51,6 +52,7 @@ public data class User(
     var devices: List<Device> = mutableListOf(),
     var online: Boolean = false,
     var createdAt: Date? = null,
+    val deactivatedAt: Date? = null,
```

Comment:
It is nice to have the fields next to each other, but there is no need to break the API for users that don't use named params. Since it has a default value, it should be at the end, will have a smaller impact.

Reply:
Updated according to your suggestion.